nx/test/nx/defn/grad_test.exs

```diff
@@ -509,6 +509,29 @@ defmodule Nx.Defn.GradTest do
     end
   end
 
+  describe "inverse hyperbolic functions" do
+    defn grad_arcsinh(t), do: grad(t, Nx.arcsinh(t))
+    defn grad_arccosh(t), do: grad(t, Nx.arccosh(t))
+    defn grad_arctanh(t), do: grad(t, Nx.arctanh(t))
+
+    test "computes gradient of inverse hyperbolic functions" do
+      for _ <- @iters do
+        t = Nx.random_uniform({}, -100.0, 100.0, type: {:f, 64})
+        check_grads!(&Nx.arcsinh/1, &grad_arcsinh/1, t, eps: 0.1)
```

Comment:
I assume a lower eps fails frequently?

Reply:
I think it's fine to lower it or remove it to use the default value.